lib/database.js

```diff
@@ -2,24 +2,41 @@ const config = require('../knexfile')[process.env.NODE_ENV || 'development']
 const knex = require('knex')(config)
 
 module.exports = {
+  getAllQuestions,
   getQuestionById,
   getAnswersById,
   addQuestion,
   addAnswer
 }
 
-function getQuestionById () {
-  // knex
+function getAllQuestions () {
+  return knex('questions')
+    .select()
+    .catch((error) => console.log(error))
+
+}
+
+function getQuestionById (id) {
+  return knex('questions')
+    .where('id', id)
+    .catch((error) => console.log(error))
```

Comment:
I would be cautious about this... I think it might swallow your error so that the response doesn't send anything back to the client. We'll let it slide for now though ;)